Userland/Libraries/LibGfx/ImageFormats/TGALoader.cpp

```diff
@@ -43,6 +43,23 @@ static_assert(sizeof(TGAHeader) == 18);
 
 }
 
+class TGAImageIdent : public Gfx::Metadata {
```

Comment:
nit: s/ident/identifier/ in class name, field, etc?

Reply:
Done :+1: